### ISSUE TITLE
Request: Accessible Docs

### ISSUE THREAD

**Issue by mixmix** [entropyxyz/entropy-core#1110]
JS is trying to read the docs on the latest release. We need to understand an API change so that we know what is expected / safe / good, and what is an error state.

Problems
- can't see any online docs 
- can't build your docs
   - README : `cargo doc --no-deps --open` Mix + Frankie both hit bugs. We each spent 5 mins trying to resolve.

We would love to be able to focus on the issues at hand instead of debugging docs so we can debug an API change
:heart: :pray:

**Comment by HCastano** [comment]
@mixmix Online docs can be found on docs.rs. Here is the [link for the `entropy-tss@0.3.0-rc.1`](https://docs.rs/entropy-tss/0.3.0-rc.1/entropy_tss/index.html) crate for example.

Would you be able to provide the errors you're getting while building the docs locally? I was able to build them fine on my end.

As far as the docs themselves, admittedly they're not the most up to date. We do recognize this (see https://github.com/entropyxyz/entropy-core/issues/1065) and is something we're gonna improve over time.

**Comment by mixmix** [comment]
Thanks @HCastano , I think links out to that site from the [README](https://github.com/entropyxyz/entropy-core/blob/master/README.md) could be great - it's one of those things which is probably obvious for Rust devs but when the docs just say "build the docs" we assumed that was the only path (after not finding much on docs.entropy.xyz)

**Comment by mixmix** [comment]
Hitting rounds of errors I progressed by installing (on Ubuntu):
- openssl
- libssl-dev
- protobuf-compiler

Got stuck at:


```bash
cargo doc --no-deps --open
   Compiling librocksdb-sys v0.11.0+8.1.1
   Compiling sc-chain-spec v31.0.0
   Compiling im-rc v15.1.0
   Compiling pulldown-cmark v0.9.6
    Checking mixnet v0.7.0
    Checking wasmtime-environ v12.0.2
   Compiling portable-atomic v1.6.0
    Checking clap_builder v4.5.19
   Compiling sc-mixnet v0.8.0
   Compiling sc-network-sync v0.34.0
   Compiling sp-runtime v32.0.0
   Compiling wasm-opt-sys v0.116.0
    Checking pallet-staking v29.0.2
   Compiling wasmtime-fiber v12.0.2
    Checking pallet-parameters v0.3.0-rc.1 (/home/NAME/projects/ENTROPY/core/pallets/parameters)
   Compiling console v0.15.8
   Compiling prost v0.12.4
    Checking sp-mixnet v0.5.0
   Compiling ittapi-sys v0.3.5
    Checking rtoolbox v0.0.2
   Compiling memoffset v0.9.1
error: failed to run custom build command for `librocksdb-sys v0.11.0+8.1.1`

Caused by:
  process didn't exit successfully: `/home/NAME/projects/ENTROPY/core/target/debug/build/librocksdb-sys-4ac176bbf944160f/build-script-build` (exit status: 101)
  --- stderr
  rocksdb/include/rocksdb/c.h:65:10: fatal error: 'stdarg.h' file not found
  thread 'main' panicked at /home/NAME/.cargo/registry/src/index.crates.io-6f17d22bba15001f/librocksdb-sys-0.11.0+8.1.1/build.rs:40:10:
  unable to generate rocksdb bindings: ClangDiagnostic("rocksdb/include/rocksdb/c.h:65:10: fatal error: 'stdarg.h' file not found\n")
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

**Comment by JesseAbram** [comment]
Have you installed the pre requisite https://stackoverflow.com/questions/59169661/cant-compile-nearcore-project-after-last-rocksdb-version-bump

**Comment by mixmix** [comment]
Thanks @JesseAbram it was missing the substrate deps ... and the `wasm-pack`

Managed to put that together through your pointer and reading the github actions. SWEET that you have an action that tests the docs build. LOVE IT

Here's a PR which locks this learning in: https://github.com/entropyxyz/entropy-core/pull/1124/files

**Comment by HCastano** [comment]
Closed via #1124.
